packages/core/src/components/tabs/tabPanel.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2024 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import classNames from "classnames";
+import * as React from "react";
+
+import { AbstractPureComponent, Classes, Utils } from "../../common";
+
+import { type TabProps } from "./tab";
+import { generateTabIds, type TabTitleProps } from "./tabTitle";
+
+export interface TabPanelProps extends Pick<TabProps, "className" | "id" | "panel">, Pick<TabTitleProps, "parentId"> {
```

Comment:
I wonder if we should also support the `renderActiveTabPanelOnly` prop here and do an early return when `isHidden` as a convenience without needing to set that up yourself and to keep the same feature set offered by uncontrolled tabs

Reply:
But it wouldn't hurt anything to add your reccomendation as a bonus 🤷

Reply:
done https://github.com/palantir/blueprint/pull/6896/commits/fc9d945ef2bf844c84c1fdd73ee4357d8dae76d6